src/site/content/en/blog/floc/index.md

```diff
@@ -17,8 +17,6 @@ feedback:
   - api
 ---
 
-## Summary
-
 FLoC provides a privacy-preserving mechanism for interest-based ad selection.
```

Comment:
Consider mentioning the full name here if not in the title. People will probably wonder the acronym stands for.